package.json

```diff
@@ -79,6 +79,8 @@
     "husky": "^8.0.1",
     "jest": "^26.6.3",
     "jest-config": "^26.6.3",
+    "jsdom": "20.0.0",
```

Comment:
The following dependencies are probably not needed as well: jsdom, jsdom-global and sinon

Reply:
removed the unneeded dependecies